cobbler/remote.py

```diff
@@ -579,13 +587,21 @@ def _log(self, msg, user=None, token=None, name=None, object_id=None, attribute=
         msg = "REMOTE %s; user(%s)" % (msg, m_user)
 
         if name is not None:
+            if not isinstance(name, str):
+                return
+            if not validate_obj_name(name):
+                return
             msg = "%s; name(%s)" % (msg, name)
 
         if object_id is not None:
+            if not validate_obj_id(object_id):
+                return
             msg = "%s; object_id(%s)" % (msg, object_id)
 
         # add any attributes being modified, if any
         if attribute:
+            if not (isinstance(attribute, str) and attribute.isidentifier()) or keyword.iskeyword(attribute):
```

Comment:
```suggestion
            if (isinstance(attribute, str) and attribute.isidentifier()) or keyword.iskeyword(attribute):
```
Identifiers and keywords should cause an early return